src/Behavior/NodeException.php

```diff
@@ -29,30 +29,36 @@ public static function create(): self
      */
     protected $domNode;
 
-    public function withDomNode(?DOMNode $domNode): self
+    /**
+     * @param DOMNode|null $domNode
+     */
+    public function withDomNode($domNode): self
     {
         $this->domNode = $domNode;
         return $this;
     }
 
     /**
-     * @deprecated since v2.1.0, use withDomNode(?DOMNode $domNode) instead
+     * @param DOMNode|null $domNode
```

Comment:
Keep the deprecations with `deprecated since v1.5.0`

Reply:
Done